packages/frontend/package.json

```diff
@@ -32,7 +32,8 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "generate:api-types": "npx openapi-typescript http://localhost:4200/api-json --output ./src/types/api-schema.ts",
-    "generate": "run-p generate:*"
+    "generate": "run-p generate:*",
```

Comment:
I don't think we want to have a command that does both generate tasks. They are both one-off tasks and not related. By adding this script we are just cluttering the package.json and making it harder for new programmers to grep

Reply:
True, it should be done postinstall